podman/api_connection.py

```diff
@@ -7,26 +7,31 @@
 from http import HTTPStatus
 from http.client import HTTPConnection
 
+import podman.containers as containers
 import podman.errors as errors
 import podman.images as images
 import podman.system as system
 
 
 class ApiConnection(HTTPConnection, AbstractContextManager):
-    """ ApiConnection provides a specialized HTTPConnection
-        to a Podman service."""
+    """ApiConnection provides a specialized HTTPConnection
+    to a Podman service."""
 
-    def __init__(self, url, base="/v1.24/libpod", *args,
-                 **kwargs):  # pylint: disable-msg=W1113
+    def __init__(
+        self, url, base="/v1.24/libpod", *args, **kwargs
```

Comment:
pylint seems unhappy with this for whatever reason, but we can resolve that later in the black usage patch.